clippy_lints/src/operators/arithmetic_side_effects.rs

```diff
@@ -184,6 +184,35 @@ impl ArithmeticSideEffects {
         }
     }
 
+    /// There are some integer methods like `wrapping_div` that will panic depending on the
+    /// provided input.
+    fn manage_method_call<'tcx>(
+        &mut self,
+        args: &[hir::Expr<'tcx>],
+        cx: &LateContext<'tcx>,
+        instance: &hir::Expr<'tcx>,
```

Comment:
IMO it would be better to use `receiver` in the method call.
https://doc.rust-lang.org/reference/expressions/method-call-expr.html#method-call-expressions
```suggestion
        receiver: &hir::Expr<'tcx>,
```

Reply:
Done